packages/components/src/mobile/link-settings/index.native.js

```diff
@@ -54,13 +54,15 @@ function LinkSettings( {
 	//	* placeholder - control component placeholder, e.g. `Add URL`
 	//	* autoFocus (url only) - whether url input should be focused on sheet opening
 	//	* autoFill (url only) - whether url input should be filled with url from clipboard
+	//	* setCursorAtStart (url only) - whether url input's selection should be set to the start. The cursor will be at the front of the link.
```

Comment:
Now that the cursor is only set to the start only when pasting a URL from the clipboard, it would be nice if we update this description and reflect that nuance. Besides, I'm wondering if we should rename the property's name, although   I'd try to avoid making it super long 😅 .

Reply:
That is a good suggestion @fluiddot Thanks for spotting 🙇🏾‍♂️